### ISSUE TITLE
Focus is not present in Dialog nested in Window after Alt + Tab

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#8363]
### Bug report

When a Dialog is opened over Kendo Window, the focused element is not persisted after Alt + Tab. 

### Reproduction of the problem
1. Open the example - https://runner.telerik.io/fullscreen/xqMSPXPe.
2. Open the Window and then the Dialog.
3. Tab to focus the second input
4. Alt+Tab out of the browser window to another application.
5. Alt+Tab back to the browser.

### Current behavior
The focus is not persistent.
In case the Dialog is not placed in a Window and the same steps are followed, the focus is persisted [dojo](https://runner.telerik.io/fullscreen/bHpYBtlb).
screencast - https://go.screenpal.com/watch/cTQqIunDnRD 

### Expected/desired behavior
The focus should be persistent when changing the open app by using Alt + Tab. 

### Environment

* **Kendo UI version:** 2025.3.825
* **Browser:** [all  ]
